packages/core/examples/checkboxExample.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
```

Comment:
nit: we typically chunk imports from other `@blueprintjs` packages and external dependencies together now:

```
import * as React from "react";
import { Checkbox, Classes } from "@blueprintjs/core";
import { BaseExample } from "@blueprintjs/docs";

// import * from "../some/relative/path/in/this/package
// import * from "../another/relative/path/in/this/package
```